<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Открытие базы</string>
  </property>
  <widget class="QLineEdit" name="addr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите адрес базы данных:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ch_base">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ch_k_f">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберете файл-ключ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Мастер-пароль от базы:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Мастер-пароль еще раз:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="view_pwd">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотреть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cls_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
